odoo/sql_db.py

```diff
@@ -257,13 +222,7 @@ class Cursor(BaseCursor):
         As a result of the above, we have selected ``REPEATABLE READ`` as
         the default transaction isolation level for OpenERP cursors, as
```

Comment:
```suggestion
        the default transaction isolation level for Odoo cursors, as
```
I think you could do this change in all the code, right? :)